packages/account/src/Styles/account.scss

```diff
@@ -1651,7 +1651,7 @@ $MIN_HEIGHT_FLOATING: calc(
             margin-inline-end: 10px;
             float: right;
 
```

Comment:
@suisin-deriv shouldnt this be   float: inline-start;

Reply:
yes it should be 🙏

Reply:
@utkarsha-deriv can you help check it in RTL see if it's looking weird 🙏

Reply:
Sure, checking it 🙏

Reply:
Should it not be float: inline-end; @suisin-deriv @amina-deriv